release_images.yml

```diff
@@ -13,6 +13,7 @@ release_images:
       python_versions: ["py37"]
       os_version: "ubuntu18.04"
       cuda_version: "cu102"
+      force_release: True
```

Comment:
Please add some more description on why this had to be forced in the PR description.

Reply:
Thanks. Updated.